index.js

```diff
@@ -27,10 +32,11 @@ class MQTTFacility extends BaseFacility {
 
     const srv = require('net').createServer(aedes.handle)
     this.server = srv
+    this.aedes = aedes
```

Comment:
what's the purpose of binding this one to class instance?

Reply:
We are accessing this from the container worker
https://github.com/PearTree-to/moria-wrk-container-bitdeer/blob/28390e8714a318a5bc67c7964eff9bf86299650f/workers/lib/worker-base.js#L66

Reply:
I can, didn't use it because it's the same instance anyways. Will update if required.

Reply:
@chetasr  @subashsn not a blocked but I think would be cleaner to use this.aedes everywhere